ManagementAPI/Private/CMAAsset.m

```diff
@@ -0,0 +1,154 @@
+//
+//  CMAAsset.m
+//  Pods
+//
+//  Created by Boris Bügling on 28/07/14.
```

Comment:
Yeah, he created the files—tbh, I don't even like these autogenerated comments at the top as it's just redundant, and useless info: the name of the file which you can see in the file system already, and the creator, who might not have even written the bulk of the contained code.